tooling/stylelint/correct-theme-token-usage/index.js

```diff
@@ -65,22 +65,19 @@ const allAllowedKeys = [
   "actionable",
   "container",
   "draggable",
-  "differential", // TODO: **deprecated:** delete here
```

Comment:
i dont think these should be removed yet or deprecated tokens may have problems?